.github/workflows/prs_check-target-branch-label.yml

```diff
@@ -1,11 +1,6 @@
 name: Check for target branch labels
 
 on:
-  pull_request_target:
-    branches:
-      - 'v*.x'
-      - 'master'
-    types: ['labeled']
```

Comment:
removing this, since it makes no sense to have it here. Needs to be on the calling action